bin/settings.ts

```diff
@@ -18,12 +18,13 @@ export const SETTINGS: OncoanalyserProps = {
   //taskMaxCpus: 256,
   pipelineMaxCpus: 2,
   taskMaxCpus: 4,
+  copyToLocalEcr: true,
   pipelineInstanceTypes: [
-    InstanceType.of(InstanceClass.R6A, InstanceSize.LARGE),
+    InstanceType.of(InstanceClass.R8G, InstanceSize.XLARGE),
```

Comment:
if this is just for benchmarking - then this is fine - but I would think at some point we would want to paramterise this somehow? So I could chose in the settings.ts whether I want ARM or x86 and it all flows through?